fission/main.go

```diff
@@ -124,6 +218,35 @@ func main() {
 				return nil
 			},
 		},
+		{
+			Name:  "get-env-variables",
```

Comment:
Two seconds is a little bit long to me. How about write related information into local config file (/home/.fission/config)?